src/Control/Lens/Internal/Indexed.hs

```diff
@@ -157,10 +154,20 @@ instance Profunctor (Indexed i) where
 #ifndef SAFE
   ( .# ) ibc _ = coerce ibc
```

Comment:
I see you did some cleanup relying on `lens`' CI using a more recent version of `hlint`, which is fine. There's another `hlint`-related simplification that I've been meaning to do for a while now involving lines like this one, where we write things like `( #. )`. There extra spaces surrounding the parentheses are quite strange, but they were necessary to work around an old `hlint` parsing bug. That bug has long been fixed, so we should be able to remove the extraneous spaces now.

Keep in mind that the line above is far from the only occurrence of this phenomenon. You don't necessarily have to perform this simplification in this PR, but if you decide not to, we should open an issue about this so that it is not forgotten.

Reply:
I'll leave this to you.

Reply:
I've done this in #933.